### PR TITLE
usagesLastModified field in ES, SQS message, and thrall consumption

### PR DESCRIPTION
**Pull request by NickPapacostas**

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala [hunk 1]
```diff
@@ -202,6 +202,7 @@ object Mappings {
           "uploadInfo" -> uploadInfoMapping,
           "suggestMetadataCredit" -> simpleSuggester,
           "usages" -> usagesMapping,
+          "usagesLastModified" -> dateFormat,
```
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala]: Why not keep this in the usages object?

Comment (NickPapacostas) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala]: We had a similar discussion around userMetadata, the idea is that usages and exports become arrays and we can't add fields to them. In the case of userMetadata it actually *is* an object so I had added it within (see https://github.com/guardian/grid/commit/a8944edff0998045b7b7c89de29eab83ef3afb98)

but for consistency between the three (exports, usages, and usermetadata) we opted to put it at the root level. Open to other approaches though :)

**Comment by theefer** [comment]
Nice :+1:

File: common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala (hunk 1 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala]: Not without completely smashing the data structure... :+1:

**Comment by theefer** [comment]
:+

**Pull request merged**